### DISCUSSION TITLE
ayo chell check this out

### DISCUSSION THREAD

**Discussion by Zenith302** [tfdsoft/famidash#167] [General]
you've got your laptop but no discord, right?
make an escargot.chat account, I'd love to chat again sometime
![image](https://github.com/famidash/famidash/assets/84691508/9b23f9e1-7434-40f0-8693-345bf4d213c1)

**Comment by glacierjl** [comment]
yo what about aim phoenix

**Comment by zephyrside** [comment]
real

**Comment by zephyrside** [comment]
tried installing; linux doesn't like it
oh well :shrug:
